galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -371,3 +371,9 @@ messages:
     args:
       - name: annotation
         type: string
+
+  - name: "GatewayDuplicateCertificate"
+    code: IST0137
+    level: Error
+    description: "Duplicate certificate in multiple gateway, causing 404"
+    template: "Duplicate certificate in multiple gateway, causing 404."
```

Comment:
can we put which certificate/gateway caused it?

Reply:
done, putting gateway which caused it